frontend_build/src/webpack.config.base.js

```
@@ -79,8 +79,14 @@ var config = {
           preserveWhitespace: false,
           loaders: {
             js: 'buble-loader',
-            stylus: combineLoaders(vueStylusLoaders),
-            scss: combineLoaders(vueSassLoaders),
+            stylus: ExtractTextPlugin.extract({
+              fallback: 'style-loader',
```

Should this be `vue-style-loader`?

Conversely, might be worth omitting it if we don't expect it to be used? I.e. better to have it fail hard - I could see this potentially leading to missed issues because styles happened to work in Chrome but weren't actually getting processed correctly.

---

Yes - although, you're right, and we should get rid of the fallback, and use `allChunks: true` instead (the default, false causes CSS in extracted chunks to not be bundled into CSS).